rstream/client.py

```diff
@@ -172,9 +173,34 @@ async def start(self) -> None:
         self._conn = Connection(self.host, self.port, self._ssl_context)
         await self._conn.open()
         self.start_task("listener", self._listener())
+
         self.add_handler(schema.Heartbeat, self._on_heartbeat)
         self.add_handler(schema.Close, self._on_close)
 
+    async def run_queue_listener_task(self, subscriber_name: str, handler: HT[FT]):
+
+        if subscriber_name not in self._frames:
+            self.start_task(
+                "run_delivery_handlers" + subscriber_name,
+                self._run_delivery_handlers(subscriber_name, handler),
+            )
+
+    async def _run_delivery_handlers(self, subscriber_name: str, handler: HT[FT]):
+
+        while True:
+            frame_entry = None
+            try:
+                frame_entry = await self._frames[subscriber_name].get()
+                maybe_coro = handler(frame_entry)
+                if maybe_coro is not None:
+                    await maybe_coro
+            except Exception as e:
+                if frame_entry is not None:
+                    logger.exception("Error while handling %s frame ", str(frame_entry.__class__))
+                else:
+                    logger.exception("Error while handling a frame " + str(e))
```

Comment:
nit: `logger.exception` attaches the error with it's traceback to the message, so there's no need for `+ str(e)`

Reply:
OK!